src/main/resources/entity-types/circulation/simple_loan_policy.json5

```diff
@@ -273,6 +262,32 @@
       queryable: true,
       visibleByDefault: false,
       valueGetter: ":sourceAlias.jsonb->'loansPolicy'->>'closedLibraryDueDateManagementId'",
```

Comment:
The property in the DB ends in `Id`, so I guess we should rename this in the translations as "...management type UUID" right? 😛